src/DSE.Open.Results/ResultStatus.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Down Syndrome Education International and Contributors. All Rights Reserved.
+// Down Syndrome Education International and Contributors licence this file to you under the MIT license.
+
+namespace DSE.Open.Results;
+
+public enum ResultStatus
```

Comment:
Unspecified maps to Ok iff no notifications error or above, else BadRequest?

Reply:
I am in two minds about merging logic - e.g. if you have a `Found` but then get a `ServerError` from a later operation, presumably the error trumps the found.

Let's flow the change and see how it fits in actual use.